### PR TITLE
Setup requirements and installation instructions

### PR DESCRIPTION
**Pull request by jfischoff**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+diffusers
+numpy
+optax
+torchvision
+datasets
+Pillow
+wandb
+tqdm
+huggingface_hub
+accelerate
```
Comment (bguisard) [inline requirements.txt:15]: I noticed that a few packages from `requirements_flax.txt` are missing here, but I don't know whether we'd use them or not.

```
chex
orbax
ftfy
tensorboard
Jinja2
```

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+jax
```
Comment (bguisard) [inline requirements.txt:1]: In the community events repo they pinned jax and flax versions as follows:
```
jax[tpu]==0.4.5
flax==0.6.7
```

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,15 @@
+jax 
+jaxlib
+flax
+torch
+transformers
+diffusers
+numpy
+optax
+torchvision
+datasets
```
Comment (bguisard) [inline requirements.txt:10]: I think we'll also want to add `webdataset`, but I can do that when we update the training script.

File: x/test-train-unet-frame-scratch [hunk 4]
```diff
@@ -0,0 +1 @@
+python src/train-unet-from-scratch.py
```
Comment (bguisard) [inline x/test-train-unet-frame-scratch:1]: Is this file needed?

**Review by bguisard — COMMENTED** [review]
Looks good for a first commit, but I haven't tested the training script yet.

I think it may be a good idea to set up a `.gitignore` to keep unwanted files out of the repo.

**Comment by jfischoff** [comment]
> Looks good for a first commit, but I haven't tested the training script yet.
> 
> I think it may be a good idea to set up a `.gitignore` to keep unwanted files out of the repo.

There should be one I think?

File: requirements.txt (hunk 1 above)
Comment (jfischoff) [reply requirements.txt:15]: Let's leave them out for now and add them if we get errors. I don't no a better way, but having unnecessary libs can lead to more conflicts down the road.
